src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -30,6 +30,7 @@
 #include "gc/shared/barrierSetAssembler.hpp"
 #include "gc/shared/barrierSetNMethod.hpp"
 #include "interpreter/interpreter.hpp"
+#include "jvmci/jvmci_globals.hpp"
```

Comment:
I would suggest #if INCLUDE_JVMCI for the added header file.

Reply:
Thanks for your comment. Agree. Will add it.